kolibri/plugins/learn/assets/src/views/LibraryPage/AccessibleChannelCard.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+
+  <KCard
+    :to="to"
+    layout="horizontal"
+    thumbnailDisplay="small"
+    thumbnailAlign="right"
+    :title="title"
+    :headingLevel="headingLevel"
+  >
+    <template #thumbnailPlaceholder>
+      <div>
+        <ChannelThumbnail
+          :thumbnail="contentNode.thumbnail"
+          style="width: 100px"
+        />
+      </div>
+    </template>
+    <template #belowTitle>
+      <div>
+        <KTextTruncator
+          :text="numberOfResources"
+          :maxLines="1"
+        />
+        <KTextTruncator
+          :text="contentNode.description"
+          :maxLines="2"
+        />
+      </div>
+    </template>
+  </KCard>
+
+</template>
+
+
+<script>
+
+  import ChannelThumbnail from './../ChannelThumbnail.vue';
+
+  export default {
+    name: 'AccessibleChannelCard',
+    components: {
+      ChannelThumbnail,
+    },
+    props: {
+      to: {
+        type: Object,
+        required: true,
+      },
+      title: {
+        type: String,
+        required: true,
+      },
+      headingLevel: {
+        type: Number,
+        required: false,
+        default: 2,
+      },
+      contentNode: {
+        type: Object,
+        required: true,
+      },
+    },
+    computed: {
+      numberOfResources() {
+        return `${this.contentNode.num_coach_contents} ${this.$tr('resources')}`;
+      },
+    },
+    $trs: {
```

Comment:
@marcellamaki would you provide guidance on where to place this string? I believe we're already moving away from in-component `$trs` but am not clear myself on where to put this one in. Thank you.

Reply:
It looks like we have a `resourcesLabel` : 'Resources' in `commonCoreStrings` so let's update this to use that string. (Also saves us a translation!) 

For other strings we might run into for the cards, @AlexVelezLl  was just assigned [this issue, which will create a shared file in `kolibri-common` for additional strings](https://github.com/learningequality/kolibri/issues/12609) that may help for anything that will be used for searching/filtering/resources in shared components for this project. Once that's merged, we can place any strings there.

Reply:
Oh I see, thanks! Wasn't aware of that.